packages/mobile/src/components/scrubber/Slider.tsx

```diff
@@ -282,6 +283,24 @@ export const Slider = memo(
       }
     }, [isPlaying, play, pause, mediaKey, durationRef])
 
+    const appState = useAppState()
+
+    /**
+     * Ensures the slider handle's position is correctly updated when app
+     * becomes active.
```

Comment:
This looks right! we do similar in stems on window focus
https://github.com/AudiusProject/stems/blob/master/src/components/Scrubber/hooks.ts#L88